util/module_storage_test.go

```diff
@@ -2,24 +2,37 @@ package util
 
 import (
 	"io/ioutil"
+	"os"
 	"testing"
 
+	"github.com/spf13/afero"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestDownloadModule(t *testing.T) {
+	a := assert.New(t)
 	dir, e := ioutil.TempDir("", "fogg")
 	assert.Nil(t, e)
 
-	d, e := DownloadModule(dir, "./test-module")
-	assert.Nil(t, e)
-	assert.NotNil(t, d)
-	assert.NotEmpty(t, d)
+	pwd, e := os.Getwd()
+	a.NoError(e)
+
+	fs := afero.NewBasePathFs(afero.NewOsFs(), pwd)
+	d, e := DownloadModule(fs, dir, "github.com/chanzuckerberg/fogg-test-module")
```

Comment:
does this mean you need network to be able to run tests now? do we want to add something like a `// +build integration` tag to files that require network?